src/Illuminate/Mail/TransportManager.php

```diff
@@ -97,8 +97,8 @@ protected function createMailDriver()
      */
     protected function createMailgunDriver()
     {
-        $client = new HttpClient;
         $config = $this->app['config']->get('services.mailgun', []);
+        $client = new HttpClient(Arr::get($config, 'guzzleOptions', []));
```

Comment:
well, options like this are confusing to people because nobody's ever sure if they need to use camelCase or snake_case.